core/templates/domain/community_dashboard/FeaturedTranslationLanguageObjectFactory.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating and mutating instances of frontend
+ * featured translation language domain objects.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+export interface FeaturedTranslationLanguageBackendDict {
+  'language_code': string;
+  'description': string;
```

Comment:
Updated for description, but if we do this for language_code we get `Identifier 'language_code' is not in camel case.eslintcamelcase`

Reply:
Interesting. I guess since the backend dict is intended to be used in the Python backend code, we need it to be in snake_case, and keeping the field names as strings avoids the linter warning in somewhat of a hacky way. Then, it's probably fine to leave it as strings.

Reply:
Done